packages/vapor/scss/elements/color-dot.scss

```diff
@@ -7,6 +7,7 @@
     min-width: var(--size);
     height: var(--size);
     min-height: var(--size);
+    margin: auto 0;
```

Comment:
Not sure I understand this change. I seems that with the suggested solution, the color dot is lower than what we had before.

#### Problem reported
![image](https://user-images.githubusercontent.com/35579930/124028045-8b111480-d9c1-11eb-8cc2-7b6eb8719159.png)

#### Suggested solution
![image](https://user-images.githubusercontent.com/35579930/124028117-9f551180-d9c1-11eb-817f-2d398ee1bd97.png)

#### Before (without `margin: auto 0;`)
![image](https://user-images.githubusercontent.com/35579930/124028165-ae3bc400-d9c1-11eb-95b7-94443f44dbb5.png)

Reply:
This work is only part of the solution, the rest is done in the UI admin and it is nearing completion.

I should have put this PR in draft, a thousand excuses